airflow/plural/docs/basic-auth.md

```diff
@@ -0,0 +1,95 @@
+## Using the Airflow API
+
+To use the [Airflow Stable REST API](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html) 
+follow the steps below
+
+### edit values.yaml
+
+You'll need edit `airflow/helm/airflow/values.yaml` in your installation repo to enable basic auth to the REST API:
+
+```yaml
+airflow:
+  airflow:
+    airflow:
+      config:
+        AIRFLOW__API__AUTH_BACKENDS: airflow.api.auth.backend.basic_auth
+```
+
+Alternatively, you should be able to do this in the configuration section for airflow in your plural console as well.
+
+### create an Airflow User
+
+Create an Airflow user that will be used to make the API calls. To do this, you'll need to identify your Airflow 
```

Comment:
we can perhaps add docs on how to do this in the console (since you can exec into pod containers there even easier)

Reply:
@michaeljguarino added option to use plural console